go.mod

```
@@ -156,4 +154,3 @@ replace github.com/gin-gonic/gin => github.com/gin-gonic/gin v1.7.0
 
 replace github.com/cosmos/cosmos-sdk/db => ./db
 
```

```suggestion
replace github.com/tecbot/gorocksdb => github.com/cosmos/gorocksdb v1.2.0
```
Should use the cosmos fork here, the changes on my fork are merged in there

---

we don't need replace any more - in `master` we already migrated fully to `cosmos/gorocksdb`.